Explorer/Assets/Scripts/Global/AppArgs/ApplicationParametersParser.cs

```diff
@@ -53,7 +52,7 @@ private void ParseApplicationParameters(string[] cmdArgs)
                 {
                     if (arg.Length > 2)
                     {
-                        lastKeyStored = arg.Substring(2);
+                        lastKeyStored = arg[2..];
```

Comment:
Personally I prefer `Substring` syntax, maybe we should standardise it in our guidelines?

Reply:
modern C# approach is to use range operator, and it could be a bit better optimized on the compiler side. Of course performance gain would be neglectable in this scenario, but if we go with standardization, then  better to use this one rather than good-old `Substring`.